docs/recommendations.rst

```diff
@@ -0,0 +1,81 @@
+response to those investigations.
+
+The Carbon Black Cloud SDK for Python offers assistance for dealing with Recommendations.
+
+Getting the List of Recommendations
+-----------------------------------
+
+By querying the ``Recommendation`` object, you can see which recommendations have already been generated for you by
+the Carbon Black Cloud.
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> api = CBCloudAPI(profile='sample')
+    >>> from cbc_sdk.endpoint_standard import Recommendation
+    >>> query = api.select(Recommendation).set_statuses(['NEW', 'ACCEPTED', 'REJECTED']).sort_by('impact_score', 'DESC')
+    >>> recslist = list(query)
+    >>> for rec in recslist:
+    ...     print(f"{rec.recommendation_id} - {rec.workflow_.status} - {rec.impact_.impact_score}")
+    ...
+    5e6926d4-0c55-4757-a94d-e05883d5ee4c - ACCEPTED - 1.1710311
+    c9221b98-f64c-45dc-acb4-93caacd9dcee - ACCEPTED - 0.71904415
+    8afdcebc-12ca-4b63-8d8f-2d8055b652f2 - NEW - 0.6051892
+    100503cd-1897-425f-93b5-1ccba320438d - ACCEPTED - 0.2678737
+    b44586ea-4bba-4684-addb-d4934c10b3e2 - NEW - 0.1663951
+    3f89a837-034c-4b81-9f4c-f673a36ccb5c - NEW - 0.11177378
+    d9a9a122-8440-44d2-9fa4-84e77345155b - ACCEPTED - 0.08829709
+    7fcb6092-7adb-4069-861b-fed439d5f22a - REJECTED - 0.063399866
+    26ddb565-aff6-4b68-895c-fc286aa5f101 - ACCEPTED - 0.05499694
+    908e691e-5dbf-4028-bb52-a41cedada2f9 - ACCEPTED - 0.016786003
+
+**N.B.:** If you do not set status values on the recommendation query with ``set_statuses()``, the search defaults to
+looking for ``NEW`` recommendations *only.*
+
+Recommendations Workflow
+------------------------
+
+Individual recommendations in the ``NEW`` state may be accepted or rejected by calling their ``accept()`` or
+``reject()`` methods, respectively.
+
+::
+
+    # Continued from first example
+    >>> sublist = [rec for rec in recslist if rec.workflow_.status == 'NEW']
+    >>> sublist[0].accept('Comment for acceptance')
+    >>> print(sublist[0].workflow_.status)
+    ACCEPTED
+    >>> sublist[1].reject('Comment for rejection')
+    >>> print(sublist[1].workflow_.status)
+    REJECTED
+
+Individual recommendations in the ``ACCEPTED`` or ``REJECTED`` states may be reverted to the ``NEW`` state by calling
+their ``reset()`` method.
+
+::
+
+    # Continued from first example
+    >>> sublist = [rec for rec in recslist if rec.workflow_.status == 'REJECTED']
+    >>> sublist[0].reset()
+    >>> print(sublist[0].workflow_.status)
+    NEW
+
+Recommendations and Reputation Overrides
+----------------------------------------
+
+A recommendation in the ``ACCEPTED`` state will have a reputation override created for it.  You can retrieve that
+object with the ``reputation_override()`` method.
+
+::
+
+    # Continued from first example
+    >>> sublist = [rec for rec in recslist if rec.workflow_.status == 'ACCEPTED']
+    >>> reputation_override = sublist[0].reputation_override()
+    >>> print(reputation_override.id)
```

Comment:
Here too, printing the reputation override internal ID is a good start (if I just want to catalogue this for later), but it would be even better (especially for demonstration code like this, that helps me as a reader understand what the code's actually doing) to show the override's metadata - the list, type, description and value

Reply:
Again, this is just an _example._ Besides, the details of reputation overrides are beyond the scope of this guide, which deals with _recommendations._

Reply:
I would disagree Reputation Overrides is a key detail of Recommendations. The guide doesn't need to be specifically targeted instead it should focus on the entire experience of the feature

Reply:
You could also change the print to be `>>> print(reputation_override)` as I believe we have `__str__` functions in the base class which does a better job of showing more of the objects details

Reply:
It's likely that people will copy directly from a guide to try something so having valuable content is key

Reply:
> You could also change the print to be `>>> print(reputation_override)` as I believe we have `__str__` functions in the base class which does a better job of showing more of the objects details

Yeah, I saw that, but the output looked ugly, so I switched to just printing the ID.

Reply:
If the output looked ugly then does that need fixed for this specific object? We can override the `__str__` method

Reply:
No, I just think the way we output the objects is kind of ugly to put in example pages. But I see Reputation Override includes at least one on its page, so I'll change over to doing that if it'll let this page get past review.

Reply:
If I'm following the updated content correctly, looks like it's now "calling" print(reputation_override) and showing a pretty-printed columnular-ish display of all the properties of the outcome of accepting the recommendation - which is awesome. Thank you!